crates/cargo-util-schemas/src/manifest/mod.rs

```diff
@@ -1787,5 +1787,5 @@ pub struct UnresolvedError;
 fn dump_manifest_schema() {
     let schema = schemars::schema_for!(crate::manifest::TomlManifest);
     let dump = serde_json::to_string_pretty(&schema).unwrap();
-    snapbox::assert_data_eq!(dump, snapbox::file!("../../manifest.schema.json"));
+    snapbox::assert_data_eq!(dump, snapbox::file!("../../manifest.schema.json").raw());
```

Comment:
Should we add a round-trip check for serialization here?

Something like this?

```rust
serde_json::from_str::<crate::manifest::TomlManifest>(&std::fs::read_to_string("mainfest.schema.json").unwrap()).unwrap();
```

Reply:
I feel like that won't generally buy us much.  If we did anything, it should probably be checking the schema against valid manifests.